dist/scripts/services/SongPlayer.js

```diff
@@ -35,9 +35,9 @@
             
             currentBuzzObject = new buzz.sound(futureSong.audioUrl, {
                 formats: ['mp3'],
-                preload: true    
+                preload: true
             });
-            
+            currentBuzzObject.setVolume(SongPlayer.volume);
```

Comment:
I put this in to have the initial volume set to 80, so there would be a default initial volume start.